internal/command/action.go

```diff
@@ -200,3 +217,32 @@ func returnStatusOfError(err error) (int, error) {
 	}
 	return 0, err
 }
+
+// printProviderVersions returns a string of all provider versions
+func printProviderVersions()(string, error){
+	var providerVersions bytes.Buffer
+	
+	providers, err := prov.GetAllProviders()
+	if err != nil {
+		return "", err
+	}
+
+	for _, provider := range providers {
+		version, err := exec.Command(path.Join(prov.DefaultPath, provider), "--version").Output()
+		if err != nil {
+			providerVersions.WriteString(fmt.Sprintf("%s: unknown version\n", provider))
+			continue
+		}
+
+		//removes trailing newline character(s)
+		if version[len(version)-1] == 10{
+			version = version[0:len(version)-1]
+			if version[len(version)-1] == 13{
+				version = version[0:len(version)-1]
+			}
+		}
```

Comment:
https://golang.org/pkg/strings/#TrimSuffix

Reply:
So at this point `version` is a []byte. I'm chopping off the last index if it is a `/\n` or `/\r`. `version` doesn't end up getting converted to a string until:
```
providerVersions.WriteString(fmt.Sprintf("%s version %s\n", provider, version))
```
where it is being appended to the return string. I'm worried this may make it more complex if I did it this way, because I'll have to convert it to a string, trim, then append. What do you think?

Reply:
nah it should be fine. in fact the main change would be that you're just converting it to a string earlier and making the code easier to read at a realistically inconsequential increase in RAM allocation (string vs []byte).

Reply:
Just did this